build/Makefile

```diff
@@ -495,23 +487,38 @@ push-allocator-image: $(ensure-build-image)
 build-allocator-image: $(ensure-build-image) build-allocator-binary build-licenses build-required-src-dist
 	docker build $(agones_path)/cmd/allocator/ --tag=$(allocator_tag) $(DOCKER_BUILD_ARGS)
 
+# build list of images to include in the manifest for the sidecar image.
+push_manifest = $(sidecar_linux_amd64_tag)
 ifeq ($(WITH_WINDOWS), 1)
+push_manifest += $(foreach windows_version, $(WINDOWS_VERSIONS), $(sidecar_tag)-windows_amd64-$(windows_version))
+endif
+ifeq ($(WITH_ARM64), 1)
+push_manifest += $(sidecar_linux_arm64_tag)
+endif
+
 # push the gameservers sidecar image
-push-agones-sdk-image: push-agones-sdk-linux-image
+push-agones-sdk-image: push-agones-sdk-linux-image-amd64
+ifeq ($(WITH_WINDOWS), 1)
+push-agones-sdk-image: push-agones-sdk-windows
+endif
+ifeq ($(WITH_ARM64), 1)
+push-agones-sdk-image: push-agones-sdk-linux-image-arm64
+endif
 	# Ensure that the sidecar manifest is removed before creating a new one.
 	-DOCKER_CLI_EXPERIMENTAL=enabled docker manifest rm $(sidecar_tag)
+	# Agones should start using the -linux_amd64 suffix for Linux images. This allows multi-arch support in the future.
```

Comment:
Haven't we done this? I think this comment (which was relocated) should be removed instead.

Reply:
Good catch